app/scripts/services/fixture-loader-service.js

```diff
@@ -23,29 +23,32 @@ angular.module('lmisChromeApp')
      * @returns {*}
      */
     var loadDatabaseFromRemote = function(dbName) {
```

Comment:
this would be
```
var loadDatabaseFromRemote = retriable(function(dbName) {
      return pouchStorageService.getRemoteDB(dbName)
        .then(function(db){
            return db.allDocs();
         })
        .then(function(res) {
             var data = res.rows;
                  var dbRecords = [];
                  for (var i in data) {
                    var record = data[i].key;
                    dbRecords.push(record);
                  }
                  return utility.castArrayToObject(dbRecords, 'uuid');
        });
```